src/aero_model.F90

```diff
@@ -604,6 +602,81 @@ end subroutine aero_model_emissions
   ! private methods
   !=============================================================================
 
+  subroutine surf_area_dens( ncol, mmr, pmid, temp, beglev, endlev, sad, reff, sfc )
+    ! djlo : diam kept out as argument
+    use mo_constants     , only : pi
+
+    ! dummy args
+    integer,  intent(in)  :: ncol
+    real(r8), intent(in)  :: mmr(:,:,:)
+    real(r8), intent(in)  :: pmid(:,:)
+    real(r8), intent(in)  :: temp(:,:)
+    ! real(r8), intent(in)  :: diam(:,:,:) ! djlo : kept out as argument
```

Comment:
I would prefer to remove this comment (i.e., the whole line) but if you keep it, please also modify it:
```suggestion
    ! real(r8), intent(in)  :: diam(:,:,:) ! diam, used by MAM, is not used in OSLO_AERO
```

Reply:
I have kept the line and modified it as suggested.